pkg/controller/adam/adam.go

```diff
@@ -196,9 +199,26 @@ func (adam *Ctx) ConfigGet(devUUID uuid.UUID) (out string, err error) {
 	return adam.getObj(path.Join("/admin/device", devUUID.String(), "config"), mimeProto)
 }
 
-// CertsGet get attest certs for devID
-func (adam *Ctx) CertsGet(devUUID uuid.UUID) (out string, err error) {
-	return adam.getObj(path.Join("/admin/device", devUUID.String(), "certs"), mimeJSON)
+// GetECDHCert get cert for ECDH exchange for devID
+func (adam *Ctx) GetECDHCert(devUUID uuid.UUID) ([]byte, error) {
```

Comment:
What does ECDH stands for?

Reply:
https://en.wikipedia.org/wiki/Elliptic-curve_Diffie%E2%80%93Hellman